src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -813,9 +816,6 @@ export default class DefaultAudioVideoController
     }
     this.sessionStateController.perform(SessionStateControllerAction.FinishDisconnecting, () => {
       if (!reconnecting) {
-        // Do a hard reset of the signaling client in case this controller is reused;
-        // this will also cause `this.meetingSessionContext` to be reset if reused.
-        this.meetingSessionContext.signalingClient = null;
```

Comment:
Consolidating on the cleanup functions. When i added this to work around stale values of `videoSubscriptions` i didn't realize those tasks existed....